apps/andi/runtime.exs

```diff
@@ -1,7 +1,15 @@
 use Mix.Config
 
 kafka_brokers = System.get_env("KAFKA_BROKERS")
-redis_host = System.get_env("REDIS_HOST")
+get_redix_args = fn (host, password) ->
```

Comment:
It feels off that this block of code is replicated across all the configs of all the redis-using apps. Is there a better way to do this?

Reply:
Once all the apps are migrated to the umbrella, we need to think hard about how to make configuration more sane. I would argue that this block should be punted on until then.